README.md

```diff
@@ -155,6 +155,20 @@ Hub for a list of all the supported tags.
 |-------------|----------------|
 | `/var/log`  |  Log storage   |
 
+## Ports ##
+
+There are no ports exposed by this container.
```

Comment:
Doesn't this expose ports?
https://github.com/cisagov/skeleton-docker/blob/b1949f9b301eef109150dfcb1febfa376e3997d8/Dockerfile#L35
https://github.com/cisagov/skeleton-docker/blob/b1949f9b301eef109150dfcb1febfa376e3997d8/docker-compose.yml#L23-L27

Reply:
I think it's just showing how to expose a port and the subsequent docker-compose-fu. I do not believe it has a functional purpose.

Reply:
See commit 1f6d090.

Reply:
Great catch @mcdonnnj! 👍